### PR TITLE
Add Tutorials to the documentation.

### PR DESCRIPTION
**Pull request by aterzis-google**

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -90,6 +90,7 @@ Read more about this in :doc:`advanced/jit`.
    notebooks/Logistic_Regression
    notebooks/Custom_Networks
    examples
+   tutorials
```
Comment (AlexeyKurakin) [inline docs/source/index.rst:93]: `notebooks/*` sections are also called tutorials, so after adding this line table of contents will look like following:

Getting Started

* Installation and Setup
* Objax Basics Tutorial
* Logistic Regression
* Creating Custom Networks for Multi-Class Classification
* Code Examples
* Tutorials

...

-------------------

I think maybe this **tutorials** item should be named `Additional tutorials` to distinguish from basic tutorials which we wanted user to read first.

File: docs/source/tutorials.rst [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Tutorials
```
Comment (AlexeyKurakin) [inline docs/source/tutorials.rst:1]: As mentioned above, maybe rename it to "Additional Tutorials"

File: docs/source/index.rst (hunk 1 above)
Comment (aterzis-google) [reply docs/source/index.rst:93]: Changed the title of tutorial.rst to "Additional tutorials"

File: docs/source/tutorials.rst (hunk 2 above)
Comment (aterzis-google) [reply docs/source/tutorials.rst]: Done

**Comment by aterzis-google** [comment]
PTAL

**Review by AlexeyKurakin — APPROVED** [review]

**Pull request merged**